packages/core/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@sentry/core",
+  "version": "0.1.0",
+  "description": "Sentry core implementation for all JavaScript related SDKs",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "typescript": {
+    "definition": "dist/index.d.ts"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "@types/jest": "^21.1.2",
+    "@types/node": "^8.0.32",
+    "jest": "^21.2.1",
+    "prettier": "^1.8.0",
+    "ts-jest": "21.2.2",
+    "ts-node": "^3.3.0",
+    "tslint": "^5.8.0",
+    "tslint-config-prettier": "^1.6.0",
+    "tslint-eslint-rules": "^4.1.1",
+    "typescript": "2.6.1",
+    "vrsource-tslint-rules": "^5.8.0"
+  },
+  "engines": {
+    "node": ">=6.9.5 <9.0.0",
+    "npm": ">=3.10.7 <6.0.0",
+    "yarn": ">=1.0.2 <2.0.0"
+  },
+  "scripts": {
+    "clean": "rm -rf ./dist",
+    "dist": "npm run clean && tsc -p tsconfig.json",
+    "test": "npm run dist && jest",
+    "test:watch": "jest --watch --notify"
+  },
+  "jest": {
```

Comment:
By default, jest applies the `"jsdom"` environment which essentially polyfills some browser APIs. However, this means that even though your tests pass, the code might not run in an actual node process. However, since you want to run it in both the browser and node, it might make sense to run tests in both environments and use the [`--env`](https://facebook.github.io/jest/docs/en/cli.html#env-environment) option to switch between them. If you don't want to do this, declare `"testEnvironment": "node"` here as this is more of a common demeanor. 

Example for running both (needs `npm-run-all` as devDependency):

```json
{
  "dist": "npm run clean && tsc -p tsconfig.json",
  "test:browser": "jest --env=jsdom",
  "test:node": "jest --env=node",
  "test": "npm-run-all dist test:browser test:node"
}
```